### ISSUE TITLE
Support for markdown links

### ISSUE THREAD

**Issue by danieltomasz** [shaunabanana/intrigue#7]
**Is your feature request related to a problem? Please describe.**

I am using markdown links to link between my projects, papers
This is especially handy on MacOS with Hook app, but is possible to do on other platform too
For example I use Zotero translator to get markdown links to papers in my Zotero bibliography manager, for example
```
[Parameterizing neural power spectra into periodic and aperiodic components](zotero://select/items/1_28VX6C9T)
```
Currently this doesnt render as a link

![image](https://user-images.githubusercontent.com/7980381/170844047-1535c2ea-2eeb-470f-b362-2bf7e7e7a4bd.png)

**Describe the solution you'd like**

```
[Parameterizing neural power spectra into periodic and aperiodic components](zotero://select/items/1_28VX6C9T)
```
Should be displayed as a link and be clickable (even if with ctrl or cmd key)
for example  if I would add a link to this thread to a intrigue canvas, it should render the same way as here in text: 
[New Issue · shaunabanana/intrigue](https://github.com/shaunabanana/intrigue/issues/new?assignees=&labels=&template=feature_request.md&title=)
not 
```
[New Issue · shaunabanana/intrigue](https://github.com/shaunabanana/intrigue/issues/new?assignees=&labels=&template=feature_request.md&title=)
```

**Describe alternatives you've considered**

**Additional context**

**Comment by shaunabanana** [comment]
Hi @danieltomasz!

Thank you for your suggestion! Your use of the Hooks app is very inspiring, as it solves a problem in the current workflow where there's no backlink to Zotero. 

The good news is I've already implemented links in the web version currently under development. It is natively supported by TipTap, so adding it was pretty simple. In the mean time, I'll try to push a new release these days in this version as well.

**Comment by danieltomasz** [comment]
there is also a Zotero translator that outputs markdown link to Zotero URI of item https://github.com/silentdot/zotero-markdown-translator

When set in settings, It could be used together with Zutilo or the even Vanilla Zotero to copy or drag items from Zotero

**Comment by danieltomasz** [comment]
@shaunabanana  Great it do what intended! 
2 thing I can report:
1) with the new release it seems that I cannot access preferences (I am on M1 mac)
2) Is possible to make option to mark the node containing Zotero URI link as blue the same way as this is done for other references?
`Enter an DOI, ISBN, or URL into a note to have it automatically turned into a "reference note" (blue).`

**Comment by shaunabanana** [comment]
Thanks for the quick response! If you mean the "Preferences" menu item in the app menu, it might be an accidental addition that wasn't meant to be there. I believe currently there's no preferences implemented in Intrigue, but it is planned in the future. Sorry for the confusion!

As for the reference part, unfortunately it might take some time to implement. Currently the links are created directly by TipTap editor, and therefore bypasses my reference identifier detection code. I'll see if I can do something about it in the next release!

**Comment by danieltomasz** [comment]
Dont worry! Thanks again  for your hard work!

If may discuss one more thing here I think the possibility to have linking between notes in the app menu would be useful, after installing I was confused  and have a problem to learn how to do that and only reading the README.md help me learning how to link notes 

If this will be easier you might consider allowing setting color of the  note (maybe even having a  predefined set like in zotero, I posted about it here https://github.com/shaunabanana/intrigue/issues/8,  but I understood this might take  some work to implement too)

**Comment by shaunabanana** [comment]
I've just published a new pre-release for v1.0☺️ In the new version, links are correctly turned into references (albeit being a bit too aggressive. Will fix in the actual release). Linking can now be done by clicking a button, so it should be more visible now. Therefore, I'll close this issue for now :)
